stage1/init/pod.go

```diff
@@ -168,10 +160,10 @@ func (p *Pod) appToSystemd(ra *schema.RuntimeApp, interactive bool) error {
 	}
 
 	env := app.Environment
-	env.Set("AC_APP_NAME", name)
+	env.Set("AC_APP_NAME", appName.String())
```

Comment:
@yifan-gu sorry I am not following - which part of on disk store are you talking about? The CAS should not be affected here surely?